lib/oxidized/model/stoneos.rb

```diff
@@ -0,0 +1,38 @@
+class Stoneos < Oxidized::Model
+#Hillstone Networks StoneOS software
```

Comment:
This project enforces a style that requires the comments to be at the same nesting level as the rest of the block (thus, either the comment goes before the class definition, or should be 2 spaces deep), and a space between the comment character and the contents of the comment, thus `# Hillstone Networks StoneOS software`